tests/helpers/ingest.py

```diff
@@ -0,0 +1,72 @@
+import json
+import logging
+from io import BytesIO
+from typing import Any
+
+logger = logging.getLogger(__name__)
+
+
+default_collection = {
+    "import_id": "test.new.collection.0",
+    "title": "Test title",
+    "description": "Test description",
+}
+
+
+def custom_collection(updates: dict[str, Any]) -> dict[str, Any]:
+    new_collection = default_collection.copy()
+    new_collection.update(updates)
+    return new_collection
```

Comment:
You can also use spread syntax here e.g.
```suggestion
    new_collection = default_collection.copy()
    new_collection.update(updates)
    return {**default_collection, **updates}
```

Which makes me wonder if we could just do that in the test e.g.
```python
{**default_collection, "import_id": "test.new.collection.1"}
```
?

Reply:
Interesting, I'll give it a go